rosbag2_performance/writer_benchmarking/src/writer_benchmark.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020, Robotec.ai sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "rmw/rmw.h"
+#include "rosbag2_storage/serialized_bag_message.hpp"
+#include "std_msgs/msg/byte_multi_array.hpp"
+#include "rosbag2_transport/storage_options.hpp"
+
+#include "writer_benchmark.hpp"
+
+using namespace std::chrono_literals;
+
+WriterBenchmark::WriterBenchmark()
+  : rclcpp::Node("writer_benchmark")
+{
+  this->declare_parameter("frequency");
```

Comment:
You can directly set the default values: 
```suggestion
  this->declare_parameter("frequency", 100);
```

Reply:
same below for all `declare_parameter` calls.